portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,18 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function randomPik() {
+  const imageIndex = Math.floor(Math.random() * 6) + 1;
+  const imgUrl = 'images/suprisedimgs/sup' + imageIndex + '.jpg';
```

Comment:
You can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here.